pyproject.toml

```diff
@@ -38,7 +38,7 @@ gvm-cli = "gvmtools.cli:main"
 gvm-script = "gvmtools.script:main"
 
 [tool.poetry.dependencies]
-python = "^3.9"
+python = "^3.9.1"
```

Comment:
Are you sure this is correct? As I understand this and the dependabot warning, this dependency would still include the vulnerable version.

Reply:
No I am not. As far as I understand the issue dependabot can't update the crypo dependency because we allow 3.9. If we are requiring 3.9.1 or above dependabot will be able to resolve the dependency issue. But not sure if that is correct.